wandb/integration/sagemaker/resources.py

```diff
@@ -26,6 +26,6 @@ def parse_sm_resources():
         )
     conf = json.load(open(sm_files.SM_RESOURCE_CONFIG))
     if len(conf["hosts"]) > 1:
-        run_dict["group"] = os.getenv("TRAINING_JOB_NAME")
+        run_dict["run_group"] = os.getenv("TRAINING_JOB_NAME")
```

Comment:
Whats the difference between group and run_group?

Reply:
the params in `run_dict` used to be entered into kwargs and then applied to settings via `_apply_init`, which then passes through this (https://github.com/wandb/client/blob/master/wandb/sdk/wandb_settings.py#L1040). now we apply it via `_apply_setup` which doesn't have this param subbing thing (which looks like it was originally a hack to reconcile user facing vs internal param keys)

Reply:
I see, nice! And there's no other params that could come in via sagemaker params that need to be subbed?

Reply:
nope looks like that's all we're putting in there for now